yb-voyager/cmd/templates/assessmentReport.template

```diff
@@ -263,38 +263,47 @@
             </table>
         {{end}}
 
+        {{if not $hasUnsupportedFeatures}} <!-- Check if no unsupported features were found -->
+            <p>No unsupported features were present among the ones assessed.</p>
+        {{end}}
+
         {{ if .MigrationCaveats}}
-            <h2>Migration caveats</h2>
-            <p></p>
-            {{range .MigrationCaveats}}
-                {{if .Objects}} <!-- Check if Objects is not empty -->
-                    {{if .DisplayDDL }} <!-- for these feature we are displaying the DDLs-->
-                        <h4>{{.FeatureName}}</h4>
-                        <p>{{.FeatureDescription}}</p>
-                        <div class="scrollable-div">
-                            <ul>
-                                {{range .Objects}}
-                                    <li class='list_item'>{{.SqlStatement}}</li>
-                                {{end}}
-                            </ul>
-                        </div>
-                    {{else}}
-                        <h4>{{.FeatureName}}</h4>
-                        <p>{{.FeatureDescription}}</p>
-                        <div class="scrollable-div">
-                            <ul>
-                                {{range .Objects}}
-                                    <li class='list_item'>{{.ObjectName}}</li>
-                                {{end}}
-                            </ul>
-                        </div>
-                    {{end}}
-                    {{if .DocsLink}}
-                        <p id="refer-p"><a href="{{.DocsLink}}" target="_blank">Details</a></p>
-                    {{end}} 
+        {{ $hasMigrationCaveats := false }}
+        <h2>Migration caveats</h2>
+        {{range .MigrationCaveats}}
+            {{if .Objects}} <!-- Check if Objects is not empty -->
+                {{ $hasMigrationCaveats = true }}
+                {{if .DisplayDDL }} <!-- for these feature we are displaying the DDLs-->
```

Comment:
please attach report screenshot of report for scenarios covered here.

Reply:
Added.